bigchaindb/lib.py

```diff
@@ -438,15 +440,22 @@ def fastquery(self):
     def get_validators(self, height=None):
         result = backend.query.get_validator_set(self.connection, height)
         validators = result['validators']
-        for v in validators:
-            v.pop('address')
-            v['voting_power'] = int(v['voting_power'])
-
         return validators
 
-    def get_validator_update(self):
-        update = backend.query.get_validator_update(self.connection)
-        return [update['validator']] if update else []
+    def get_validator_update(self, txns):
+        votes = {}
+        for txn in txns:
+            if isinstance(txn, ValidatorElectionVote):
+                election_id = txn.asset['id']
+                election_votes = votes.get(election_id, [])
+                votes[election_id] = election_votes.append(txn)
```

Comment:
You are assigning `None` to `votes[election_id]` here. In case there are two votes in the block and the second one concludes the election, this code will raise an `AttributeError`. Makes sense to add the corresponding test case.